tbump.toml

```diff
@@ -36,6 +36,18 @@ cmd = "tox -e docs||echo 'Hack so this command does not fail'"
 name = "Upgrade the pylintrc examples"
 cmd = "pylint --rcfile=/dev/null --generate-rcfile > examples/pylintrc"
 
+[[before_commit]]
```

Comment:
After the merge of the other PRs these two steps are no longer necessary as the output of the commands is now checked in the normal test suite.

Reply:
[62cf71f](https://github.com/PyCQA/pylint/pull/6529/commits/62cf71f65a435dfdb556a6650d08ece7469d861f)